app/controllers/ObligedEntitiesController.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Request, Result}
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import utils.JsonUtils.jsonFromFile
+import utils.DesResponse._
+
+import scala.concurrent.Future
+
+@Singleton()
+class ObligedEntitiesController @Inject()(headerValidator: HeaderValidatorAction)(cc: ControllerComponents)
+    extends BackendController(cc) with HeaderValidator {
+
+  def getObligedEntity(utr: String): Action[AnyContent] = headerValidator.async { implicit request =>
+    if(isUtrValid(utr)) {
+      utr match {
+        //sample
+        case "2134514321" =>
```

Comment:
is 2134514321 correct ?
Given I request a record for UTR at /trusts/obliged-entities/UTR/1234567890

Reply:
That's the test one that came with the documentation of the api. So I believe it's correct